x/concentrated-liquidity/types/gov_test.go

```diff
@@ -10,39 +10,52 @@ import (
 	"github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types"
 )
 
-func TestCreateConcentratedLiquidityPoolProposalMarshalUnmarshal(t *testing.T) {
+func TestCreateConcentratedLiquidityPoolsProposalMarshalUnmarshal(t *testing.T) {
+	records := []types.PoolRecord{
+		{
+			Denom0:             "uion",
+			Denom1:             "uosmo",
+			TickSpacing:        100,
```

Comment:
nit: we should have different tick spacings here as well & if possible another set of records (non-blocking)